gloo-edge/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+GLOO_NS=gloo-system
+TRAEFIK_NS=kube-system
```

Comment:
we should not uninstall traefik, we are not doing this for any other app. This will be made optional in future but we should not uninstall traefik.